products/memgraph/upcoming/how_to_guides/query-memgraph-programmatically.md

```diff
@@ -3,10 +3,10 @@
 ### Supported Languages
 
 If users wish to query Memgraph programmatically, they can do so using the
-[Bolt protocol](https://boltprotocol.org). Bolt was designed for efficient
-communication with graph databases and Memgraph supports
-[Version 1](https://boltprotocol.org/v1) of the protocol. Bolt protocol drivers
-for some popular programming languages are listed below:
+[Bolt protocol](https://boltprotocol.org/). Bolt was designed for efficient
```

Comment:
I think we can remove https://boltprotocol.org/ and just link to https://7687.org/
> The current version is Protocol Version 1.
is on the old page...